packages/components/src/components/domainproperties/issues/models.ts

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2020 LabKey Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Record } from 'immutable';
+
+import { DomainDesign, DomainField } from '../models';
+
+export class IssuesModel extends Record({
+    exception: undefined,
+    domain: undefined,
+    entityId: undefined,
+    createdBy: undefined,
+    created: undefined,
+    modifiedBy: undefined,
+    modified: undefined,
+    containerId: undefined,
+    domainId: undefined,
+    name: undefined,
+    singularItemName: undefined,
+    pluralItemName: undefined,
+    commentSortDirection: undefined,
```

Comment:
I think we want this one to have a default value of 'ASC' to match the existing GWT behavior. Similarly, do we want default values for other props like singularItemName, pluralItemName, etc. to match the GWT behavior?

Reply:
Good point, now done